src/desktop/__tests__/ui/components/list.test.jsx

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import { ListComponent as List } from 'ui/components/List';
+
```

Comment:
The snapshot test may fail when run in an environment that uses a different date format (e.g. MM/DD/YYYY or DD.MM.YYYY). We should consider mocking `moment` to resolve this